licenses.html

```diff
@@ -4210,6 +4210,7 @@ <h4>Used by:</h4>
                     <li><a href=" https://github.com/bluss/indexmap ">indexmap</a></li>
                     <li><a href=" https://github.com/rust-itertools/itertools ">itertools</a></li>
                     <li><a href=" https://github.com/dtolnay/itoa ">itoa</a></li>
+                    <li><a href=" https://github.com/dtolnay/itoa ">itoa</a></li>
```

Comment:
this one can be removed, `itoa` is already there
```suggestion   
```

Reply:
this is generated automatically and the CI step tests that the file did not change. Itoa appears two times because the dependency tree imports two different versions

Reply:
(Probably something we can make a PR elsewhere to fix?)

Reply:
3  different pull requests :grin:
serde_json pulls in itoa 1.0, while 0.4 is requested by csv, http (the version released last week gets 1.0), hyper, and serde_urlencoded

Reply:
ouch